packages/react-components/react-avatar/src/components/AvatarGroup/useAvatarGroup.tsx

```diff
@@ -53,19 +53,26 @@ export const useAvatarGroup_unstable = (props: AvatarGroupProps, ref: React.Ref<
     required: true,
     defaultProps: {
       children: overflowButtonChildren,
+      type: 'button',
     },
   });
 
   const overflowContent = resolveShorthand(props.overflowContent, {
     required: true,
     defaultProps: {
-      'aria-label': 'Overflow',
       children: overflowChildren,
       role: 'list',
       tabIndex: 0,
```

Comment:
my one comment would be I don't think we need the list to be focusable anymore, since it's now wrapped within the popup surface.

Since it looks like the popover surface is the one that now has `overflow: scroll`, I think that's the only node that would need to be focusable for keyboard scroll

Reply:
Otherwise it looks good!

Reply:
That would bring the same problem of trapFocus not working correctly since the children of PopoverSurface is what will be focused, not the PopoverSurface. I could add the `overflow: scroll` to the PopoverContent so that it can be focused and allow the user to scroll. What do you think about that change?